Loader/SymfonyFixturesLoader.php

```diff
@@ -72,6 +79,61 @@ protected function createFixture($class)
         return $this->loadedFixtures[$class];
     }
 
+
+    /**
+     * Returns the array of data fixtures to execute.
+     *
+     * @return array $fixtures
+     */
+    public function getFixtures()
+    {
+        $fixtures = parent::getFixtures();
+
+        if ($this->tag) {
+            $mapping = $this->fixtureTagMapping;
+            $tag = $this->tag;
+            $fixtures = array_filter(
+                $fixtures,
+                function ($fixture) use ($mapping, $tag) {
+                    return isset($mapping[$fixture]) && $mapping[$fixture] === $tag;
+                },
+                ARRAY_FILTER_USE_KEY
+            );
+        }
+
+        return $fixtures;
+    }
+
+    /**
+     * Creates an array of the fixtures and there tags
+     *
+     * @param array $services
+     *
+     * @return void
+     */
+    public function addFixtureTagMapping(array $services)
+    {
+        foreach ($services as $service => $tags) {
+            foreach ($tags as $attributes) {
+                if (array_key_exists('set', $attributes)) {
+                    $this->fixtureTagMapping[$service] = $attributes['set'];
```

Comment:
Come to think of it, you will need to support multiple services for the same tag too.

So I guess you will need a nested array anyway.

Reply:
yes thats what i tought, but i asked this in the description.
I can't pass an array as attribute.
Do i just pass a string and explode it ? on what character ?

Reply:
I was referring to `$this->fixtureTagMapping`, that can be an array of arrays.

Where do you need to have an array it the attribute? I do not follow you there.

Reply:
Ok, I think I get it. You cannot have multiple "set" values in one tag. You have to use multiple tags for the same service instead. I guess that is a known drawback with the tagging system in the dependency container.

Reply:
You do not need it. If someone wants multiple "set" tags for the same service, they can just add multiple tags for that service, with one set attibute in each tag.